cl_dll/saytext.cpp

```diff
@@ -280,7 +281,7 @@ void CHudSayText :: SayTextPrint( const char *pszBuf, int iBufSize, int clientIn
 	m_iFlags |= HUD_ACTIVE;
 
 	if (m_HUD_saytext_sound->value != 0.0f)
-		PlaySound( "misc/talk.wav", 1 );
+		PlaySound(m_HUD_saytext_sound_path->string, m_HUD_saytext_sound->value);
```

Comment:
this is a breaking change, what if someone has 10 in their config

Reply:
> what if someone has 10 in their config

Literally nothing:


https://user-images.githubusercontent.com/58108407/157049967-1cbb39b0-ab40-481d-ae66-ac06a596ab57.mp4